fastlane/Deliverfile

```diff
@@ -1,17 +0,0 @@
-require 'dotenv'
-Dotenv.load('~/.wpios-env.default')
-
-screenshots_path "./fastlane/promo-screenshots/"
-app_identifier "org.wordpress"
-
-# Make sure to update these keys for a new version
-app_version "17.7"
-
-skip_binary_upload true
-overwrite_screenshots true
-phased_release true
-precheck_include_in_app_purchases false
-
-api_key_path ".configure-files/app_store_connect_fastlane_api_key.json"
-
-app_rating_config_path "./fastlane/metadata/ratings_config.json"
```

Comment:
👍  Just to say I checked that each setting that was in this deleted `Deliverfile` was now mirrored back in the `Fastfile`, either in the [common settings constant](https://github.com/wordpress-mobile/WordPress-iOS/pull/16805/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R909-R917) or for each app-specific call, and can confirm that all of them were there, so I think we're indeed safe to delete this now 🎉